index.js

```diff
@@ -30,7 +37,7 @@ const DEFAULT_HEADERS = {
   'Sec-Fetch-Mode': 'navigate',
   'Sec-Fetch-User': '?1',
   'Sec-Fetch-Dest': 'document',
-  'Referer': 'https://evisaforms.state.gov/acs/make_calendar.asp?CSRFToken=AD406CCFF7524F7383E13726D05205CD&nMonth=10&nYear=2021&type=1&servicetype=06&pc=SNJ',
+  'Referer': 'https://evisaforms.state.gov/acs/make_calendar.asp?CSRFToken=AD406CCFF7524F7383E13726D05205CD&nMonth=10&nYear=2021&type=1&servicetype=06&pc=' + EMBASSY,
```

Comment:
please use strings templates consistency, rather than concatenation here.

Reply:
I tried that at first but wasn't able to make it work, probably because of the order of when constants are defined. Will make another attempt when I have time to learn javascript better. 

FYI, all that really matters is the const url string in getCookieAndCSRFToken.

Reply:
> I tried that at first but wasn't able to make it work, probably because of the order of when constants are defined. Will make another attempt when I have time to learn javascript better.

I'm not quite following you. Either way, I've merged your changes, [and made a new commit to use string templates + formalizing the embassy mappings](https://github.com/k-funk/us-cr-embassy-passport-appointment-scraper/commit/b33a0ba88169264266e7dcbe1435ffe458d7ebbf).

I'm also going to rename this repo since it's not just `us-cr-...`

Thanks for your contribution!